src/react/ColorPicker/components/HueSpectrum.tsx

```diff
@@ -58,24 +56,25 @@ export default function HueSpectrum(props: HueSpectrumProps) {
         }   
     }
 
-    function initContainerEl() {
```

Comment:
Should not remove this function, instead rename and use it later in handleMouseUp, handleMouseMove... to avoid repetition.